lib/foreman_maintain/hammer.rb

```diff
@@ -0,0 +1,37 @@
+module ForemanMaintain
+  class Hammer
```

Comment:
What about turning the hammer into a feature? I think it would fit a bit better to the whole concept